src/test/operators/table_scan_test.cpp

```diff
@@ -283,4 +302,101 @@ TEST_F(OperatorsTableScanTest, ScanOnReferenceSegmentWithNullValue) {
   }
 }
 
+TEST_F(OperatorsTableScanTest, ScanWithNotComparableSearchValue) {
+  auto scan_1 = std::make_shared<TableScan>(_table_wrapper, ColumnID{0}, ScanType::OpGreaterThanEquals, "Hello");
```

Comment:
```suggestion
  const auto scan_1 = std::make_shared<TableScan>(_table_wrapper, ColumnID{0}, ScanType::OpGreaterThanEquals, "Hello");
```

I assume all scans below could also be `const`, but not too sure. Also, it would break consistency with the other test cases, so you decide 🌚